helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -408,6 +408,9 @@ agent:
     # so that agent pods can be discovered via dns etc
     enabled: true
 
+    # (optional) Override the default service internal traffic policy
+    internalTrafficPolicy: "Local"
```

Comment:
I think it would be good to add information about available options like it is done [here](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/helm-charts/splunk-otel-collector/values.yaml#L182) for `cloudProvider`